int8 my_id
--- 
bool recived